snc-library.sh

```diff
@@ -281,7 +281,7 @@ function wait_till_cluster_stable() {
     done
 
     # Wait till all the pods are either running or complete state
-    while oc get pod --no-headers --all-namespaces | grep -v Running | grep -v Completed; do
+    while ${OC} get pod --no-headers --all-namespaces | grep -v Running | grep -v Completed; do
```

Comment:
Feel free to split this change in a separate PR, this could be merged independently from the arm support.

Reply:
removed.

Reply:
opened https://github.com/code-ready/snc/pull/458